config/clients/java/template/README_retries.mustache

```diff
@@ -0,0 +1,24 @@
+By default API requests are retried up to 15 times on 429 and 5xx errors and credential requests are retried up to 15 times on 429 and 5xx errors. In both instances they will wait a minimum of 100 milliseconds between requests.
+
+In order to change the behavior for API requests, when using the `ClientConfiguration` builder, call `maxRetries` to control the amount of retries and `minimumRetryDelay` to control the minimum wait time between retried requests.
+
+```java
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dev.openfga.sdk.api.client.OpenFgaClient;
+import dev.openfga.sdk.api.configuration.ClientConfiguration;
+import java.net.http.HttpClient;
+
+public class Example {
+    public static void main(String[] args) throws Exception {
+        var config = new ClientConfiguration()
+                .apiUrl(System.getenv("FGA_API_URL")) // If not specified, will default to "https://localhost:8080"
```

Comment:
Do we really default to: `https://localhost:8080`? 🤔 

If yes, it should
1) be changed to `http://localhost:8080`
2) made similarly in other SDKs

(this is to be resolved later in another issue, not this one)

Reply:
That's what the codeblock I copied this from says 😄  I'll review and see

Reply:
Fixed this and the other incorrect references in the initializing section